CIP-0035/README.md

```diff
@@ -156,26 +181,22 @@ So, for example, a single CIP could propose multiple new builtin functions, or a
 
 Proposals for additions to the set of Plutus Core builtins SHOULD be proposed in a CIP and SHOULD adhere to the following additional process.
 
-In order to move to Draft status, it MUST include:
+In order to move to Proposed status, it MUST include:
 - In the Specification:
     - Names and types/kinds for the new functions or types.
     - A source for the implementation (e.g. a library which can be linked against); or a generic description of the functionality which is implementable in any programming language.
-    - In the Rationale:
+    - For new types: a precise description of the measure used for the size of a value of that type.
+    - For new builtin functions: a costing function for the builtin function.
+- In the Rationale:
     - An argument for the utility of the new builtins.
+    - If an external implementation is provided: an argument that it is trustworthy.
+    - Discussion of how any measures and costing functions were determined.
 
 It SHOULD also include:
 - In the Rationale
     - Examples of real-world use cases where the new additions would be useful.
     - A comparison with an implementation using the existing features, and an argument why the builtin is preferable (e.g. better performance).
```

Comment:
In some cases an implementation using the existing features might be difficult or impossible. For example, Implementing a complex cryptographic function using the basic integer operations might be a very large amount of work for something that would clearly be inferior to a C implementation.

Maybe something like "If possible/feasible, a comparison ..."?